src/components/Dashboard/CourtStats.js

```diff
@@ -1,20 +1,152 @@
 import React from 'react'
-import { Box, Distribution } from '@aragon/ui'
+import { Box, GU, textStyle, useTheme } from '@aragon/ui'
+import useCourtStats from '../../hooks/useCourtStats'
+import { formatUnits } from '../../lib/math-utils'
+import {
+  useANJBalanceToUsd,
+  useTokenBalanceToUsd,
+} from '../../hooks/useTokenBalanceToUsd'
+import Loading from './Loading'
 
-const CourtStats = () => {
+const splitAmount = amount => {
+  const [integer, fractional] = amount.split('.')
   return (
-    <Box heading="Court Stats">
-      <Distribution
-        heading="Disputes"
-        items={[
-          { item: 'New', percentage: 37 },
-          { item: 'Adjudicated', percentage: 22 },
-          { item: 'Appeals', percentage: 15 },
-          { item: 'Executed', percentage: 12 },
-        ]}
-      />
+    <span
+      css={`
+        margin-right: 5px;
+      `}
+    >
+      <span className="integer">{integer}</span>
+      {fractional && (
+        <span
+          css={`
+            font-size: 16px;
+          `}
+        >
+          .{fractional}
+        </span>
+      )}
+    </span>
+  )
+}
+
+function CourtStats() {
+  const theme = useTheme()
+  const [stats, fetching] = useCourtStats()
+
+  return (
+    <Box heading="Court Metrics" padding={3 * GU}>
+      {(() => {
```

Comment:
In this case, it might be simpler as:

```js
{fetching ? (
  <Loading height={86} />
): (
  stats.map((stat, index) => {
    ...
  })
)}

Reply:
for me is way more cleaner to read as it is now, what do yo think?

Reply:
Let's leave it as is!